backend/src/main/scala/com/softwaremill/adopttapir/starter/StarterService.scala

```diff
@@ -26,23 +25,9 @@ class StarterService(projectGenerator: ProjectGenerator, filesManager: FilesMana
                 generatedFiles <- formattedGeneratedFiles
                 _ <- filesManager.createFiles(tempDir, generatedFiles)
                 zippedFile <- filesManager.zipDirectory(tempDir)
-                _ <- increaseMetricCounter(starterDetails)
+                _ <- IO(Metrics.increaseGenerationMetricCounter(starterDetails))
```

Comment:
`increaseGenerationMetricCounter` => `increaseZipGenerationMetricCounter`? We could generate preview as well, so adding "zip" make this more explicit.

Reply:
done, I have also renamed `increasePreviewMetricCounter` to `increasePreviewOperationMetricCounter` so that it is more explicit